packages/harness/package.json

```diff
@@ -10,7 +10,7 @@
 		"./package.json": "./package.json"
 	},
 	"peerDependencies": {
-		"@angular/cdk": "^13.3.2 || ^14.0.0-rc.0 || ^15.0.0-rc.0 || ^16.0.0-rc.0 || ^17.0.0"
+		"@angular/cdk": ">=13.3.2"
```

Comment:
Thanks for changing it to a minimal version, which makes it little more friendly for next upcoming Angular versions, but the change isn't applied in the lock file yet.

Reply:
Ops, forgot to run `yarn`, should be good now.